test/node/download-tracker-magnet.js

```diff
@@ -15,7 +16,7 @@ test('Download using HTTP tracker (via magnet uri)', function (t) {
 })
 
 function magnetDownloadTest (t, serverType) {
-  t.plan(10)
+  t.plan(11)
```

Comment:
Okay, so this was a timing thing. Basically now `noPeers` is getting emitted twice because the tracker gets sent a 'started' then a 'completed' message and both times there are no peers in the response. I changed the `.on('noPeers')` to `.once('noPeers')` so this will be more reliable.